crates/orchestrator/src/workers/update_state.rs

```diff
@@ -11,15 +11,20 @@ use crate::workers::Worker;
 
 pub struct UpdateStateWorker;
 
+/// Worker that creates a new state transition job for the next set of blocks to be settled.
 #[async_trait]
 impl Worker for UpdateStateWorker {
     async fn run_worker(&self, config: Arc<Config>) -> color_eyre::Result<()> {
         tracing::trace!(log_type = "starting", category = "UpdateStateWorker", "UpdateStateWorker started.");
+        // A single single state transition job can process a contiguous set of blocks.
 
+        // gets latest state transition job that was registered in the database
```

Comment:
few of the comments here are not needed. Ig variable names are self explanatory.

Reply:
removed.